view/frontend/templates/content/widget/sliders/list.phtml

```diff
@@ -0,0 +1,66 @@
+<?php /** @var \Blackbird\ContentManager\Block\Content\Widget\ContentList $block */ ?>
+<?php
+    $collection = $block->getCollection();
+    $link = $block->getLink();
+?>
+<div class="block widget contenttype-listing contents-grid grid">
+    <?php if (!empty($block->getTitle())): ?>
+        <div class="block-title">
+            <strong><?= $block->escapeHtml($block->getTitle()); ?></strong>
+        </div>
+    <?php endif; ?>
+    <div class="block-content">
+        <?php if ($collection->getSize()): ?>
+            <?php if ($block->hasPagerTop()): ?>
+                <div class="toolbar-bottom">
+                    <div class="toolbar top">
+                        <?= $block->escapeHtml($block->getPagerHtml()); ?>
```

Comment:
in Methods that return html (usually they have Html in method name) we should use escape methods. Also in this case we don't to use `/* @noEscape */` so let's remove escapeHtml here.